doc/integrations/spark.ipynb

```diff
@@ -0,0 +1,872 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "result = %sql SELECT trip_distance FROM taxi\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "data = result.toPandas()\n",
+    "\n",
+    "plt.hist(data[\"trip_distance\"])\n",
```

Comment:
Did you face any issues with `%sqlplot` instead of using `matplotlib` directly? Also, `--schema` is supported please check the docs: https://jupysql.ploomber.io/en/latest/api/magic-plot.html#sqlplot-histogram

Reply:
At the minute I've added basic functionality, for spark connection, I'll do a follow up PR to see if I can integrate more fully with some of the other functionality

Reply:
actually I've been able to get these working, just finishing up integration tests now